ccvm_simulators/solvers/dl_solver.py

```diff
@@ -257,7 +257,10 @@ def _solve(
                 Defaults to None, which generates a filename based on the problem instance name.
 
         Returns:
-            solution (Solution): The solution to the problem instance.
+            c, s (tensor): random variables 
+            c_sample, s_sample (tensor): random sample [needs attention] 
```

Comment:
Could we be more specific on these docstrings? Why `[needs attention]` is there? Is this something work in progress?

Reply:
During that time I wasn't sure how to fill out their description. Now it is fixed.